kt/godot-library/src/main/kotlin/godot/core/bridge/NativeCallable.kt

```diff
@@ -0,0 +1,230 @@
+@file:JvmName("CallableUtils")
+@file:Suppress("PackageDirectoryMismatch")
+
+package godot.core
+
+import godot.Object
+import godot.core.callable.KtCallable
+import godot.core.memory.MemoryManager
+import godot.core.memory.TransferContext
+import godot.util.VoidPtr
+import godot.util.camelToSnakeCase
+import kotlin.reflect.KCallable
+
+class NativeCallable : NativeCoreType, Callable {
+
+    constructor() {
```

Comment:
I think all those constructors should be internal. The primary way of creating a Callable (Native or Kt) should be done through the invoke methods of the parent Callable so the users don't have to care about which type it is.